examples/scenes/src/test_scenes.rs

```diff
@@ -21,26 +21,21 @@ macro_rules! scene {
             function: Box::new($name),
```

Comment:
I'd probably express this case (lines 16-22) in terms of the new addition

Reply:
Done.